### PR TITLE
This commit contains code for task6

### PR DESCRIPTION
**Pull request by Kristaps31**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,8 +13,9 @@ function App() {
       <Navbar />
       <Routes>
         <Route path="/" element={ <TopReviews />} />
-        <Route path="/reviews/:review_id" element={<IndividualReview />} />
-      </Routes>
+        <Route path="/reviews/:review_id" element={<IndividualReview /> } />
+        <Route path="/reviews/:review_id" element={ <Comments />} />
```
Comment (yalicee) [inline src/App.js]: comments should just live in Individual review

File: src/componenets/CommentCard.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import "../componenets_CSS/CommentsCard.css"
+
+const CommentCard = (comment) => {
+    console.log(comment.comment)
```
Comment (yalicee) [inline src/componenets/CommentCard.jsx]: remove log

File: src/componenets/IndividualReview.jsx [hunk 3]
```diff
@@ -14,13 +15,15 @@ function IndividualReview() {
         getReviewsById(review_id).then((game) => {
             setGameReview(game);
             setLoading(false);
+            Comments();
```
Comment (yalicee) [inline src/componenets/IndividualReview.jsx]: what is this doing?

**Review by yalicee — CHANGES_REQUESTED** [review]
Left some feedback :)

**Review by johnhutcheon — APPROVED** [review]
All approved as I've now seen this working after our ncHelp 👍

**Pull request merged**